app-creator/src/widgets/ui-map/ui-map.ts

```diff
@@ -37,106 +46,161 @@ function loadGoogleMaps(apiKey: string) {
 
 @customElement('ui-map')
 export class Map extends LitElement {
-  static styles = css`
-    #editable-view {
-      position: absolute;
-      top: 0;
-      right: 0;
-      display: flex;
-      justify-content: flex-end;
-      width: 100%;
-    }
-
-    .edit-buttons {
-      background-color: white;
-      border: var(--light-border);
-      border-radius: var(--extra-tight);
-      margin-left: var(--extra-tight);
-      cursor: pointer;
-    }
-  `;
+  static styles = css``;
 
   static attributes: AttributeMetaData = {
     // Default latitude and longitude is set to Google's Mountain View office.
     latitude: {
-      value: '39.930546488601294',
-      placeholder: '39.9305464',
+      value: '37.419857',
+      placeholder: '37.419857',
       step: 0.000000000000000000001,
       type: InputType.number,
     },
     longitude: {
-      value: '-100.825',
+      value: '-122.078827',
       step: 0.000000000000000000001,
-      placeholder: '-100.825',
+      placeholder: '-122.078827',
       type: InputType.number,
     },
     zoom: {
       value: '4',
       placeholder: '4',
       type: InputType.number,
     },
+    zoomControl: {
+      value: 'false',
+      type: InputType.select,
+      items: ['true', 'false'],
+    },
+    fullscreenControl: {
+      value: 'false',
+      type: InputType.select,
+      items: ['true', 'false'],
+    },
+    streetViewControl: {
+      value: 'false',
+      type: InputType.select,
+      items: ['true', 'false'],
+    },
+    mapTypeControl: {
+      value: 'false',
+      type: InputType.select,
+      items: ['true', 'false'],
+    },
+    mapStyles: {
+      value: 'standard',
+      type: InputType.select,
+      items: ['standard', 'silver', 'retro', 'night', 'dark', 'aubergine'],
```

Comment:
This could be done if we store the styles in the database. We would have to fetch them whenever the widget loads or when the value of the select element changes. Do we anticipate having a large number of pre-defined styles? If the number is relatively small, its easier and less costly to have them on the client. 

idea: We could have community styles so users can upload their own custom styles and make it available for all users.

Reply:
We can leave the style blobs on the client, since they are relatively light anyway. For now, I would make this list a global variable somewhere and use the constant here, so we can better keep track of the map styles and we don't need to update it in two places.